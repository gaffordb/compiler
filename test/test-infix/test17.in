let min2 : int->int =
  fun (x:int) : int -> fun (y:int) : int -> if x <= y then x else y in 
(min2 1) 2
