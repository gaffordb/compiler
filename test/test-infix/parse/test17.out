(let min2 = (fun x -> (fun y -> (if (<= x y) then x else y))) in
((min2 1) 2))
